GPSTest/src/main/res/values/arrays.xml

```diff
@@ -46,6 +46,18 @@
         <item>3</item>
     </string-array>
 
+    <string-array name="preferred_display_mode_options">
+        <item>@string/preferences_preferred_display_mode_option_dd</item>
+        <item>@string/preferences_preferred_display_mode_option_dms</item>
+        <item>@string/preferences_preferred_display_mode_option_ddm</item>
+    </string-array>
+
+    <string-array name="preferred_display_mode_options_values">
+        <item>dd</item>
```

Comment:
These strings should be defined in `do_not_translate.xml` and references via the same `@string/xxxxx` method.